<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>Login</class>
 <widget class="QWidget" name="Login" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>UtopiaPlayer - Login</string>
  </property>
  <widget class="QFrame" name="frame" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>231</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="gridLayout" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="1" column="0" >
      <widget class="QLabel" name="HostLabel" >
       <property name="text" >
        <string>Host:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="1" >
      <widget class="QCheckBox" name="RememberLogin" >
       <property name="text" >
        <string>Remember this Login</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="HostEdit" >
       <property name="maxLength" >
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="UserLabel" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>3</hsizetype>
         <vsizetype>3</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>User Name:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="LoginTypeLabel" >
       <property name="text" >
        <string>Login Type:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1" >
      <widget class="QLineEdit" name="PassEdit" >
       <property name="maxLength" >
        <number>32767</number>
       </property>
       <property name="echoMode" >
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="1" >
      <widget class="QCheckBox" name="LoginAtStart" >
       <property name="text" >
        <string>Login at Startup</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QComboBox" name="LoginTypeCombo" >
       <item>
        <property name="text" >
         <string>Anonymous Listener</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Registered Listener</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Disc Jockey</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0" >
      <widget class="QLabel" name="PassLabel" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>7</hsizetype>
         <vsizetype>7</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Password:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QLineEdit" name="UserEdit" >
       <property name="maxLength" >
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="DatabaseLabel" >
       <property name="text" >
        <string>Database:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLineEdit" name="DatabaseEdit" />
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayout" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>250</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QPushButton" name="QuitButton" >
      <property name="text" >
       <string>&amp;Quit</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="LoginButton" >
      <property name="text" >
       <string>&amp;Login</string>
      </property>
      <property name="default" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>LoginTypeCombo</tabstop>
  <tabstop>HostEdit</tabstop>
  <tabstop>DatabaseEdit</tabstop>
  <tabstop>UserEdit</tabstop>
  <tabstop>PassEdit</tabstop>
  <tabstop>RememberLogin</tabstop>
  <tabstop>LoginAtStart</tabstop>
  <tabstop>QuitButton</tabstop>
  <tabstop>LoginButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
